integration-tests/olminstall/suite/constants.py

```
@@ -94,18 +95,19 @@ def default_tests_config_path() -> Path:
-# olminstall setup-dependencies.sh args when smoke runs with operator install (Jenkins EaaS / InstallDeps).
+# olminstall setup-dependencies.sh args when smoke runs with operator install (Jenkins EPHC / InstallDeps).
 DEFAULT_SETUP_DEPENDENCIES_ARGS = "-M"
 
 # Must match ``ARTIFACT_BROWSER_URL`` / ``ARTIFACT_BROWSER_REPO_PATH`` defaults in olminstall-pipeline.yaml.
 DEFAULT_ARTIFACT_BROWSER_URL = (
     "https://app-artifact-browser.apps.rosa.konflux-qe.zmr9.p3.openshiftapps.com"
 )
 DEFAULT_ARTIFACT_BROWSER_REPO_PATH = "odh-ci-artifacts"
-# Full-matrix EaaS install + 18-component smoke,bvt exceeds Konflux default 2h (q8qpr).
-DEFAULT_OLMINSTALL_PIPELINE_TIMEOUT = "4h0m0s"
+# Full-matrix --components all smoke is sequential (~12–36m per test-* plus prepare).
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)